link/kprobe.go

```diff
@@ -49,6 +50,12 @@ type KprobeOptions struct {
 	// Can be used to insert kprobes at arbitrary offsets in kernel functions,
 	// e.g. in places where functions have been inlined.
 	Offset uint64
+	// Increase the maximum number of concurrent invocations of a kretprobe.
+	// Required when tracing some long running functions in the kernel.
+	//
+	// Deprecated: this setting forces the use of an outdated kernel API and is not portable
+	// across kernel versions.
+	RetprobeMaxActive int
```

Comment:
Should this be `uint` instead? Can you check whether the kernel parses this as signed or unsigned?

Reply:
It's `int` in kernel.  If maxactive <= 0, it is set to a default value.

Reply:
> Should this be `uint` instead? Can you check whether the kernel parses this as signed or unsigned?

It's `int` in kernel.  If maxactive <= 0, it is set to a default value.